configuration/openmrs/apps/registration/extension.json

```diff
@@ -0,0 +1,83 @@
+        "type": "link",
+        "translationKey":"REGISTRATION_SEARCH_CONFIG_KEY",
+        "url": "/search",
+        "shortcutKey": "REGISTRATION_SEARCH_CONFIG_ACCESS_KEY",
+        "icon": "fa-search",
+        "order": 1,
+        "requiredPrivilege": "View Patients"
+    },
+    "createNew":{
+        "id": "bahmni.registration.navigation.patient.create",
+        "extensionPointId": "org.bahmni.registration.navigation",
+        "type": "link",
+        "translationKey": "REGISTRATION_CREATE_NEW_CONFIG_KEY",
+        "url": "/patient/new",
+        "icon": "fa-plus",
+        "shortcutKey": "REGISTRATION_CREATE_NEW_CONFIG_ACCESS_KEY",
+        "order": 2,
+        "requiredPrivilege": "Add Patients"
+    },
+    "view":{
+        "id": "bahmni.patient.search.result.action.edit",
+        "extensionPointId": "org.bahmni.registration.patient.search.result.action",
+        "type": "link",
+        "translationKey": "REGISTRATION_VIEW_CONFIG_KEY",
+        "url": "/patient/{{patientUuid}}",
+        "icon": "fa-patient-edit",
+        "order": 1,
+        "requiredPrivilege": "View Patients"
+    },
+    "nutritionalValues":{
+        "id": "bahmni.registration.conceptSetGroup.nutritionalLevels",
+        "extensionPointId": "org.bahmni.registration.conceptSetGroup.observations",
+        "type": "config",
+        "extensionParams": {
+            "conceptName": "Nutritional Values",
+            "translationKey": "NUTRITIONAL_VALUES_LOCALE_KEY",
+            "conceptNames": ["Height", "Weight", "BMI Data", "BMI Status Data"],
+            "required":true,
+            "showLatest": true
+        },
+        "order": 1,
+        "requiredPrivilege": "Edit Visits"
+    },
+    "feeInformation":{
+        "id": "bahmni.registration.conceptSetGroup.feeInformation",
+        "extensionPointId": "org.bahmni.registration.conceptSetGroup.observations",
+        "type": "config",
+        "extensionParams": {
+            "conceptName": "Fee Information",
+            "translationKey": "FEE_INFORMATION_LOCALE_KEY",
+            "required":true
+        },
+        "order": 2,
+        "requiredPrivilege": "Edit Visits"
+    },
+    "triageLevel":{
+        "id": "bahmni.registration.conceptSetGroup.triageLevel",
+        "extensionPointId": "org.bahmni.registration.conceptSetGroup.observations",
+        "type": "config",
+        "extensionParams": {
+            "conceptName": "Haiti_Triage information",
+            "translationKey": "TRIAGE_INFORMATION"
+        },
+        "order": 2,
+        "requiredPrivilege": "Edit Visits"
+    },
+    "vitalSigns":{
+        "id": "bahmni.registration.conceptSetGroup.vitalSigns",
+        "extensionPointId": "org.bahmni.registration.conceptSetGroup.observations",
+        "type": "config",
+        "extensionParams": {
+            "conceptName": "Haiti_Vital Signs Form",
+            "translationKey": "VITAL_SIGNS"
+        },
+        "order": 1,
+        "requiredPrivilege": "Edit Visits"
+    }
```

Comment:
this should have `showLatest: true`

Reply:
[Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/5/commits/4c0ba5aea2f9cf0124c68657ea6ab904d73c5cfc)